apps/scoutgame/components/claim/components/common/TokensCell.tsx

```diff
@@ -1,10 +1,10 @@
 import { Stack, Typography } from '@mui/material';
 import Image from 'next/image';
 
-export function PointsCell({ points }: { points: number }) {
+export function TokensCell({ tokens }: { tokens: number }) {
   return (
     <Stack direction='row' alignItems='center' justifyContent='flex-end' gap={0.5}>
-      <Typography>{points}</Typography>
+      <Typography>{tokens}</Typography>
```

Comment:
pretty sure we should format this number using, eg, ceilToPrecision(tokens, 4).toLocaleString()